include/OpenSpaceToolkit/Astrodynamics/Flight/Profile.hpp

```diff
@@ -59,6 +74,48 @@ class Profile
         Custom      ///< Custom pointing mode
     };
 
+    enum class Axis
+    {
+        X,
+        Y,
+        Z
+    };
+
+    enum class TargetType
+    {
+        GeocentricNadir,
+        GeodeticNadir,
+        Trajectory,
+        Sun,
+        Moon,
+        Velocity
```

Comment:
Perhaps not worth delaying this PR, but thought: we could consider tracking the Velocity vector in ECI and Velocity vector in ECEF (which aren't pointing in the same direction, in an absolute sense, since the Earth is rotating).

Clocking against V_ECEF is for example necessary to support line scans.

Even if our implementation doesn't support this today, should our interface already embrace that case?

Reply:
Ah interesting, that's a good idea. Can atleast add the enums and a raise a ToBeImplemented error 👍

Reply:
Done!